crates/goose/src/agents/extension_manager.rs

```diff
@@ -598,12 +599,25 @@ impl ExtensionManager {
             .insert(name, Extension::new(config, client, info, temp_dir));
     }
 
-    /// Get extensions info
+    /// Get extensions info for building the system prompt.
```

Comment:
we also call this function for some reason when generating a recipe. either way, do we need to modify this at all? to me it looks like if we have code execution on, we don't even insert the extensions in the prompt (or we should).